README.rst

```diff
@@ -147,14 +147,14 @@ sub-directory:
    * (Required) taxonomy_id: taxonomy ID for organism for which genes
      are being populated;
 
-   * (Optional) systematic_col: systematic column in gene info file.
-     Default is 3;
+   * (Optional) gi_tax_id: alternative taxonomy ID for some organisms
+     (such as S. cerevisiae);
```

Comment:
It's repetitive to have documentation in README and command line help message. Consider [what we do for Manubot](https://github.com/manubot/manubot/blob/81394c0a8153fa3ad41ee3cb987227c8ea043125/README.md#process) (potentially in another PR), to have the README usage documentation come directly from the utilities help message.

Reply:
This `REAME.rst` was actually generated by sphinx based on the template in `sphinx` directory.